packages/marketplace/src/components/Modals/Settings/ConfigRow.tsx

```diff
@@ -2,15 +2,18 @@ import React from "react";
 import { Config } from "../../../types/marketplace-types";
 
 import Toggle from "../../Toggle";
-
+import SortBox from "../../Sortbox";
 const ConfigRow = (props: {
   name: string;
   storageKey: string;
   modalConfig: Config;
   clickable?: boolean;
   updateConfig: (CONFIG: Config) => void;
+  type: string | null;
+  options: string[] | null;
 }) => {
-  const toggleId = `toggle:${props.storageKey}`;
+  const type = props.type;
+  const componentId = type == "dropdown" ? "dropdown:" +props.storageKey : "toggle:" + props.storageKey;
```

Comment:
Good practice, also strict equal cuz loose equal is just funky
```suggestion
  const componentId = type === "dropdown" ? "dropdown:" + props.storageKey : "toggle:" + props.storageKey;
```